helpers/serviceHelper.js

```diff
@@ -0,0 +1,32 @@
+const chance = require('chance').Chance();
+import request from 'supertest';
+import 'dotenv/config';
+import * as constants from './constants';
+
+export const serviceData = async() => {
+  return {
+    name: chance.sentence({ words: 2 }),
+    vendor: await constants.vendorId(),
+    clientPrice: chance.prime(),
```

Comment:
Similarly to what Raf pointed out with my code, chance.prime() returns a random prime number between 2 and 10000 by default, which will not cover all scenarios